### PR TITLE
DBZ-5756 Exclude Oracle Compression Advisor tables from capture

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-5756

### CODE REVIEW

**Comment by Naros** [comment]
@roldanbob I've added a small blurb in the Oracle docs about table exclusion patterns, do you mind giving that a quick review and let me know if you'd like any wording changes.  Thanks!

File: documentation/modules/ROOT/pages/connectors/oracle.adoc [hunk 1]
```diff
@@ -1761,6 +1761,19 @@ When the {prodname} Oracle connector captures tables, it automatically excludes
...
+[id="tables-that-the-debezium-oracle-connector-excludes-when-capturing-change-events"]
+=== Tables excluded from capture
+
+When the {prodname} Oracle connector captures tables, it automatically excludes tables that match the following rules:
+
+* Compression Advisor tables matching the pattern `CMP[3|4]$[0-9]+`.
+* Index-organized tables matching the pattern `SYS_IOT_OVER_%`.
+* Spatial tables matching the patterns `MDRT_%`, `MDRS_%`, or `MDXT_%`.
+
+If a table matches any of these rules, the table will need to be renamed for the connector to capture its changes.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/oracle.adoc:1775]:
```suggestion
To enable the connector to capture a table that matches any of the preceding rules, you must rename the table.
```

**Review by prowave — APPROVED** [review]
Looks like it will solve the issue.

**Comment by jpechane** [comment]
@Naros Applied, thanks!

**Pull request merged**